spec/spec_helper.rb

```diff
@@ -18,6 +18,9 @@
 
   config.example_status_persistence_file_path = "tmp/rspec_examples.txt"
   config.order = :random
+
+  config.filter_gems_from_backtrace *%w(active_support factory_girl action_view
```

Comment:
Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication.